src/main/java/uk/gov/hmcts/reform/professionalapi/domain/OrganisationInfo.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.professionalapi.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Setter
+@Getter
+@NoArgsConstructor
+public class OrganisationInfo {
+
+    private String organisationIdentifier;
+    private OrganisationStatus status;
+    private LocalDateTime lastUpdated;
+    private List<String> organisationProfileIds;
+
+    public OrganisationInfo(String organisationIdentifier,
```

Comment:
**import lombok.AllArgsConstructor;** we can use this

Reply:
added in [a4aa031](https://github.com/hmcts/rd-professional-api/pull/1529/commits/a4aa0314c9fdcfa34d663b7aa5778296934b4e2b)